R/pkg/R/DataFrame.R

```diff
@@ -2880,7 +2880,7 @@ setMethod("fillna",
 #'
 #' @param x a SparkDataFrame.
 #' @param row.names NULL or a character vector giving the row names for the data frame.
```

Comment:
updated a few places we are referencing `NULL` literally.
there are more "null" in DataFrame or column function documentation but they are in somewhat gray area - JVM `null` are mapped to R `NA` (and not to `NULL`) - and we should look into the best way to name functions or document them.